src/main/kotlin/apply/ui/admin/selections/GradeEvaluationForm.kt

```diff
@@ -0,0 +1,62 @@
+package apply.ui.admin.selections
+
+import apply.application.GradeEvaluationData
+import apply.application.GradeEvaluationItemData
+import apply.domain.evaluationtarget.EvaluationStatus
+import com.vaadin.flow.component.select.Select
+import com.vaadin.flow.component.textfield.IntegerField
+import com.vaadin.flow.component.textfield.TextArea
+import support.views.BindingFormLayout
+
+class GradeEvaluationForm :
+    BindingFormLayout<GradeEvaluationData>(GradeEvaluationData::class) {
+
+    private val gradeEvaluationItems: MutableList<GradeEvaluationItemForm> = mutableListOf()
+    private val note: TextArea = TextArea("기타 특이사항")
+    private val evaluationStatus: Select<EvaluationStatus> = Select(*EvaluationStatus.values()).apply {
+        setItemLabelGenerator { it.label() }
+        label = "평가 상태"
+    }
+    private val sumOfScore: IntegerField = IntegerField("합계").apply { isReadOnly = true }
+
+    init {
+        add(note, evaluationStatus, sumOfScore)
+        setColspan(note, 2)
+        drawRequired()
+    }
+
+    private fun addGradeEvaluationItem(data: GradeEvaluationItemData = GradeEvaluationItemData()) {
+        val evaluationAnswerForm = GradeEvaluationItemForm().also {
+            it.fill(data)
+            it.setScoreChangeEvent { sumOfScore.value = sumOfScore() }
+            setColspan(it, 2)
+            gradeEvaluationItems.add(it)
+        }
+        addComponentAtIndex((children.count() - 3).toInt(), evaluationAnswerForm)
+    }
+
+    private fun EvaluationStatus.label() =
```

Comment:
'label' 대신 'toText'는 어떨까요?

```suggestion
    private fun EvaluationStatus.toText(): String =
```